packages/google_maps_flutter/google_maps_flutter_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.2
```

Comment:
These platform_interface changes need to be reverted so we only merge stuff pertaining to google_maps_flutter pkg in this PR (hopefully they'll get rebased away once we merge the platform_interface changes in master!)

Reply:
Done